templates/vue/src/components/TapestryApp.vue

```diff
@@ -50,25 +50,31 @@
         ></locked-tooltip>
       </svg>
     </main>
+    <div v-if="settings.renderMap" id="map-container">
+      <tapestry-map />
+    </div>
   </div>
 </template>
 
 <script>
 import DragSelectModular from "@/utils/dragSelectModular"
 import { mapMutations, mapState } from "vuex"
 import TapestryNode from "@/components/TapestryNode"
+import TapestryMap from "@/components/TapestryMap"
 import TapestryLink from "@/components/TapestryLink"
 import TapestryDepthSlider from "@/components/TapestryDepthSlider"
 import SettingsModalButton from "@/components/SettingsModalButton"
 import RootNodeButton from "@/components/RootNodeButton"
 import LockedTooltip from "@/components/LockedTooltip"
 import TapestryFilter from "@/components/TapestryFilter"
 import Helpers from "@/utils/Helpers"
+import "leaflet/dist/leaflet.css"
```

Comment:
I don't know if it's a good idea to load the leaflet css file here. What if we don't need it? That means it will load all the time even if we don't need it? Maybe a solutions like what's suggested [here](https://stackoverflow.com/questions/52767025/how-to-load-css-file-dynamically) would work. @narendrasss what do you think?